src/installer.js

```diff
@@ -44,6 +44,14 @@ var spawn = function (options, command, args, callback) {
   })
 
   spawnedProcess.on('error', function (err) {
+    if (err.name = 'ENOENT') {
+      var installer = process.platform === 'darwin' ? 'brew' : 'apt-get'
+
+      if (err.message.indexOf('fakeroot') > -1) {
+        err.message = 'Your system is missing the fakeroot package. Try ' + installer + ' install fakeroot'
```

Comment:
👍 , had it like that originally but changed syntax to better match the file's style